### ISSUE TITLE
about label/score normalization

### ISSUE THREAD

**Issue by ainnn** [SSL92/hyperIQA#7]
hey, thanks for your great work.
i viewed code and i found that there's no label normalization, e.g. normalizing scores to range [0, 1]. it's ok not to normalize when train and test on the same dataset or datasets with similar range.
in the paper, table 3 lists 3 datasets (livec, bid and knoiq) which have different score ranges. is it reasonable to use raw scores? or maybe you have normalized scores?
look forward to your reply.

**Comment by ainnn** [comment]
also, i find that the target network contains 5 fc layers in the code, while 4 fc layers claimed in the paper.

**Comment by SSL92** [comment]
Actually, we didn't do this normalization in cross database evaluation. This is because the final criterion SRCC calculates only the correlation ratio between two vectors and thus has no relationship with vector scale. 
However, it seems to be true that normalizing labels to the same scale in single database evaluation is more reasonable, since we use a fixed lr for training all databases, while it might be better to use different lr for training datasets with different label ranges.

**Comment by SSL92** [comment]
Thanks for your kind remind, there indeed exists a little difference about fc numbers from the paper and our code. However, it seems using whether 4 or 5 fc layers doesn't affect the model performance much, this is probably due to the target net has already learned sufficient quality representation in several front layers. You can also change the fc numbers by your own to see if the performance changes accordingly.

**Comment by ainnn** [comment]
Thanks for your reply.
i think it's important to normalize label to get consistent label distribution in cross database evaluation. as for using different lr for different database in single database evaluation, it does not matter because the gradient from L1 loss has nothing to do with the scale.

**Comment by SSL92** [comment]
Our pleasure ; )
